src/settings/mod.rs

```diff
@@ -13,6 +14,11 @@ use serialize;
 #[cfg(test)]
 mod test;
 
+lazy_static! {
+    static ref SQS_URL_FORMAT: Regex =
+        Regex::new("^https://sqs\\.[a-z0-9-]+\\.amazonaws\\.com/[0-9]+/[A-Za-z0-9-]+$").unwrap();
+}
```

Comment:
Let's call the existing function from the `validate` module rather than duplicating the regex inline.

Reply:
I agree with leaving this stuff inside validation, seems cleaner.

Reply:
Yeah, I agree.